update readme and example (#4)
DevExpress-Examples/obsolete-blazor-create-datagrid-at-runtime: Readme.md

<!-- default badges list -->
[![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/T848300)
[![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
<!-- default badges end -->

# DataGrid for Blazor - How to create the DataGrid and its columns at runtime

This example shows how to create the Data Grid component at runtime and generate its columns dynamically.

![Change Grid Content at Runtime](images/datagrid-change-data-on-run.gif)

> The main idea of this example is to implement a method that returns the **RenderFragment<T>** object, for example, as follows:

Reviewer: It seems strange to say that this example shows how to do something, and then say that there is another main idea. So what is this example about exactly then?

Author: The sentence before the image says what is the main idea of this example.
The sentence after the image says how this main idea can be achieved.

```suggestion
To do this, implement a method that returns the **RenderFragment<T>** object, for example, as follows:
```

Reviewer: To do this, implement...
